### PR TITLE
Convert mobile dropdown to select tag 

### PR DESCRIPTION
**Pull request by nnat425**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
   "dependencies": {
     "@nypl/dgx-header-component": "2.4.15",
     "@nypl/dgx-react-footer": "0.5.1",
+    "@nypl/dgx-svg-icons": "^0.3.8",
```
Comment (ktp242) [inline package.json]: Is dgx-svg-icons under @nypl as a public npm module now?

File: server.js [hunk 2]
```diff
@@ -48,7 +48,7 @@ app.use('/', apiRoutes);
 app.use('/', (req, res) => {
 
 // Change the page title based on having results or not. For accessibility purposes.
-const pageTitle = req.originalUrl == '/' ? 'Search NYPL.org' : "Search Results | NYPL.org";
+const pageTitle = req.originalUrl ==='/' ? 'Search NYPL.org' : "Search Results | NYPL.org";
```
Comment (ktp242) [inline server.js]: Should have a space between `===` and `/`

**Comment by ktp242** [comment]
Nicely done! But when I click the svg icon, the down arrow, it is not working.
The rest of the button works fine!

**Comment by ktp242** [comment]
I also feel we should use a real "down arrow" icon instead of using a right arrow and turn it. But if it is because we don't have available UX resource to make a new icon, I understand.

**Comment by LucasCioffi** [comment]
@ktp242 regarding the rotation of the dropdown, that's what we found in the header (screenshot).
screen shot 2018-12-03 at 6 29 08 pm

@jackkim9 you offered to give us a new SVG earlier today.  We'll take it and add it to the repo.  Thank you!
The RightWedgeIcon from https://github.com/NYPL/dgx-svg-icons is here:
`<polygon points="13 8.437 20.563 16 13 23.563 11.593 22.155 17.748 16 11.593 9.845 13 8.437" />`
We need it to turn 30 degrees clockwise.

**Comment by nnat425** [comment]
> @nnat425 @LucasCioffi
> Few notes here:
> 
> 1. I'm seeing both dropdown and tabs on desktop screen shot 2018-11-29 at 12 44 31 pm
> 2. Keyboard navigating on mobile viewport (with Willa), we were able to open the dropdown pressing the space bar but not when pressing the return key.

Issue has been fixed.

File: package.json (hunk 1 above)
Comment (nnat425) [reply package.json]: I believe so.

File: server.js (hunk 2 above)
Comment (nnat425) [reply server.js]: Fixed. Nice catch.

**Comment by ktp242** [comment]
Cool the issue of SVG icon has been fixed! Just some minor suggestions for improvements. But for your convenience I will approve now.

**Comment by ktp242** [comment]
Actually I think we really should fix the issue in package.json to avoid installing the same module twice.

**Pull request merged**